packages/flutter/README.md

```diff
@@ -7,110 +7,3 @@ focused on low-latency input and high frame rates on Android and iOS.
-
-You can always lookup an app's current locale with `Localizations.localeOf()`:
-```
-Locale myLocale = Localizations.of(context);
-```
-
-The [Localizations] widget is used to load and lookup localized values. Apps refer to the Localizations widget with `Localizations.of(context, type)`. If the device's locale changes, the Localizations widget automatically loads values for the new locale and then rebuilds widgets that used it to look up localized values. This happens because Localizations is effectively an [InheritedWidget]. When a build function refers to an inherited widget an implicit dependency on the inherited widget is created. When an inherited widget changes, its dependent contexts are rebuilt.
-
-Localized values are loaded by the Localizations widget's list of [LocalizationsDelegate]s. Each delegate must define an async `load()` method that produces an object which encapsulates a collection of values. Typically these objects provide one method per localized value.
-
-In a large app, different modules or packages might be bundled with their own localizations. That's why the Localizations widget manages a table of objects, one per LocalizationsDelegate. To retrieve the object produced by one of the LocalizationsDelegate's `load` methods, you specify a BuildContext and the object's type.  For example the localized strings for the Material widgets are defined by the `MaterialLocalizations` class. Instances of this class are created by a LocalizationDelegate provided by the `MaterialApp` class. They can be retrieved with `Localizations.of`:
-
-```dart
-Localizations.of<MaterialLocalizations>(context, MaterialLocalizations);
-```
-
-This particular `Localizations.of()` expression is used frequently, so the MaterialLocalizations class provides a convenient shortand:
-
-```dart
-static MaterialLocalizations of(BuildContext context) {
-  return Localizations.of<MaterialLocalizations>(context, MaterialLocalizations);
-}
-
-/// References to the localized values defined by MaterialLocalizations
-/// are typically written like this:
-
-tooltip: MaterialLocalizations.of(context).backButtonTooltip,
-```
-
-The example that follows can be viewed as a template for using Flutter's localizations classes. It defines a single class, MyLocalizations, that contains the app's localized resources (just one for the example). It uses the initializeMessages() function generated by the Dart intl package to load resources and Intl.message() to look them up.
-
-```dart
-class MyLocalizations {
-  MyLocalizations(this.locale);
-
-  final Locale locale;
-
-  static Future<MyLocalizations> load(Locale locale) {
-    return initializeMessages(locale.toString())
-      .then((Null _) {
-        return new MyLocalizations(locale);
-      });
-  }
-
-  static MyLocalizations of(BuildContext context) {
-    return Localizations.of<MyLocalizations>(context, MyLocalizations);
-  }
-
-  String title() => Intl.message('<title>', name: 'title', locale: locale.toString());
```

Comment:
I'd use just "Title" here, because those of us who have be traumatized by web frameworks might assume that it's some kind of markup. :-)

Also, you might consider sticking with the "cancel" example from the second paragraph in order to have a consistent example throughout.

Reply:
I think it might be better to just start with a title example; I think explaining things in terms of getters whose english translation is the same thing, foo.cancel => "Cancel" might nudge the confusometer just a little.